app/views/users/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1>List of users</h1>
\ No newline at end of file
+<h1>List of users</h1>
+<% @users.reverse_each do |user| %>
+  <%= link_to user do %>
```

Comment:
Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) 💪 Awesome!